debian/nntrainer-dev.install

```diff
@@ -13,6 +13,8 @@
 /usr/include/nntrainer/layer_context.h
 /usr/include/nntrainer/layer_devel.h
 /usr/include/nntrainer/layer_impl.h
+# custom layer kits
+/usr/include/nntrainer/app_context.h
```

Comment:
this also requires nntrainer_log.h, or you can remove the dependency.

Reply:
Yeah, I'll double check by installing this commit and if it compiles in the internal repo with including app context header

Reply:
Now verified that this is working. I removed nntrainer_log.h dependency from app_context.h